chain/index/ddls.go

```diff
@@ -0,0 +1,109 @@
+package index
+
+import "database/sql"
+
+const DefaultDbFilename = "chainindex.db"
+
+var ddls = []string{
+	`CREATE TABLE IF NOT EXISTS tipset_message (
+		id INTEGER PRIMARY KEY,
+		tipset_key_cid BLOB NOT NULL,
+		height INTEGER NOT NULL,
+		reverted INTEGER NOT NULL,
+		message_cid BLOB,
+		message_index INTEGER,
+		UNIQUE (tipset_key_cid, message_cid)
+	)`,
+
+	`CREATE TABLE IF NOT EXISTS eth_tx_hash (
+		tx_hash TEXT PRIMARY KEY,
+		message_cid BLOB NOT NULL,
+		inserted_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+	)`,
+
+	`CREATE TABLE IF NOT EXISTS event (
+		event_id INTEGER PRIMARY KEY,
```

Comment:
`id` makes sense for this one too, since it's the primary key and not foreign

Reply:
This is done.